### DISCUSSION TITLE
教程界面 https://cn.vuejs.org/tutorial/ 无法访问 Cannot access totor page

### DISCUSSION THREAD

**Discussion by AdemJensen** [vuejs-translations/docs-zh-cn#913] [General]
![image](https://github.com/vuejs-translations/docs-zh-cn/assets/23029457/3aad6933-2cdc-4bc1-bc92-5c489a7a334e)
如图所示，主要的 js 资源全部 404 导致 tutorial 界面无法访问
As shown in the picture above, all major js resources returned status 404, which prevents the access of tutorial page.

**Comment by veaba** [comment]
Cloudflare 引起，国内部分节点最近几个月不太稳定。

CTRL+ F5 强刷新下，还有问题，再多刷新几次。

暂无别的办法。
